tika/server.go

```diff
@@ -131,9 +134,8 @@ func (s *Server) Start(ctx context.Context) error {
 	for k, v := range s.JavaProps {
 		props = append(props, fmt.Sprintf("-D%s=%s", k, v))
 	}
-	propsArgs := strings.Join(props, " ")
 
-	cmd := command("java", append([]string{propsArgs, "-jar", s.jar, "-p", s.port}, s.child.args()...)...)
+	cmd := command("java", append([]string{strings.Join(props, " "), "-jar", s.jar, "-p", s.port}, s.child.args()...)...)
```

Comment:
Sorry, I probably wasn't clear. I don't think we should have spaces in the argument. Each argument should be a different element of this args slice.

Something like this, but we could do it all in one line. A little odd to have an `append` with different values on the right and left.

```suggestion
	args := append(props, "-jar", s.jar, "-p", s.port)
	args = append(args, s.child.args()...
	cmd := command("java", args...)
```